spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+feature 'user signs in', %Q{
+  As a user, I want to log in, so that I can view restricted content.
+  Acceptance Criteria:
+  * [X] - I have an option to log in.
+  * [X] - I see an error message if my log in credentials are invalid.
+  * [X] - I see my avatar if I am logged in.
+  * [X] - I have an option to edit my profile if I am logged in.
+  } do
+
+  let(:user) do
+    FactoryGirl.create(:user)
+  end
+
+  scenario 'specify valid credentials' do
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.